scapy/contrib/iec104_information_elements.py

```diff
@@ -0,0 +1,4646 @@
+        QOI_FLAG_PRIVATE_RESERVED_211: 'private reserved (211)',
+        QOI_FLAG_PRIVATE_RESERVED_212: 'private reserved (212)',
+        QOI_FLAG_PRIVATE_RESERVED_213: 'private reserved (213)',
+        QOI_FLAG_PRIVATE_RESERVED_214: 'private reserved (214)',
+        QOI_FLAG_PRIVATE_RESERVED_215: 'private reserved (215)',
+        QOI_FLAG_PRIVATE_RESERVED_216: 'private reserved (216)',
+        QOI_FLAG_PRIVATE_RESERVED_217: 'private reserved (217)',
+        QOI_FLAG_PRIVATE_RESERVED_218: 'private reserved (218)',
+        QOI_FLAG_PRIVATE_RESERVED_219: 'private reserved (219)',
+        QOI_FLAG_PRIVATE_RESERVED_220: 'private reserved (220)',
+        QOI_FLAG_PRIVATE_RESERVED_221: 'private reserved (221)',
+        QOI_FLAG_PRIVATE_RESERVED_222: 'private reserved (222)',
+        QOI_FLAG_PRIVATE_RESERVED_223: 'private reserved (223)',
+        QOI_FLAG_PRIVATE_RESERVED_224: 'private reserved (224)',
+        QOI_FLAG_PRIVATE_RESERVED_225: 'private reserved (225)',
+        QOI_FLAG_PRIVATE_RESERVED_226: 'private reserved (226)',
+        QOI_FLAG_PRIVATE_RESERVED_227: 'private reserved (227)',
+        QOI_FLAG_PRIVATE_RESERVED_228: 'private reserved (228)',
+        QOI_FLAG_PRIVATE_RESERVED_229: 'private reserved (229)',
+        QOI_FLAG_PRIVATE_RESERVED_230: 'private reserved (230)',
+        QOI_FLAG_PRIVATE_RESERVED_231: 'private reserved (231)',
+        QOI_FLAG_PRIVATE_RESERVED_232: 'private reserved (232)',
+        QOI_FLAG_PRIVATE_RESERVED_233: 'private reserved (233)',
+        QOI_FLAG_PRIVATE_RESERVED_234: 'private reserved (234)',
+        QOI_FLAG_PRIVATE_RESERVED_235: 'private reserved (235)',
+        QOI_FLAG_PRIVATE_RESERVED_236: 'private reserved (236)',
+        QOI_FLAG_PRIVATE_RESERVED_237: 'private reserved (237)',
+        QOI_FLAG_PRIVATE_RESERVED_238: 'private reserved (238)',
+        QOI_FLAG_PRIVATE_RESERVED_239: 'private reserved (239)',
+        QOI_FLAG_PRIVATE_RESERVED_240: 'private reserved (240)',
+        QOI_FLAG_PRIVATE_RESERVED_241: 'private reserved (241)',
+        QOI_FLAG_PRIVATE_RESERVED_242: 'private reserved (242)',
+        QOI_FLAG_PRIVATE_RESERVED_243: 'private reserved (243)',
+        QOI_FLAG_PRIVATE_RESERVED_244: 'private reserved (244)',
+        QOI_FLAG_PRIVATE_RESERVED_245: 'private reserved (245)',
+        QOI_FLAG_PRIVATE_RESERVED_246: 'private reserved (246)',
+        QOI_FLAG_PRIVATE_RESERVED_247: 'private reserved (247)',
+        QOI_FLAG_PRIVATE_RESERVED_248: 'private reserved (248)',
+        QOI_FLAG_PRIVATE_RESERVED_249: 'private reserved (249)',
+        QOI_FLAG_PRIVATE_RESERVED_250: 'private reserved (250)',
+        QOI_FLAG_PRIVATE_RESERVED_251: 'private reserved (251)',
+        QOI_FLAG_PRIVATE_RESERVED_252: 'private reserved (252)',
+        QOI_FLAG_PRIVATE_RESERVED_253: 'private reserved (253)',
+        QOI_FLAG_PRIVATE_RESERVED_254: 'private reserved (254)',
+        QOI_FLAG_PRIVATE_RESERVED_255: 'private reserved (255)',
+  
```

Comment:
Idem + for all the others very big lists

Reply:
Is there a reason for this request beside the number of loc?

Pro static argument: I did a quick test with a dict containing 1000 entries - the static version is 10x faster than the generated version (but we talk about 0.000028s vs 0.000221s).

Reply:
No reason other than readibility. I may be wrong, but it makes code hard to read

Reply:
You could start by generating the dicts programatically. We'll see after that what remains to do

Reply:
Before I start to change a plethora of dicts - is this an acceptable solution:
```python
    COI_FLAGS = {
        COI_FLAG_LOCAL_POWER_ON: 'local power on',
        COI_FLAG_LOCAL_MANUAL_RESET: 'manual reset',
        COI_FLAG_REMOTE_RESET: 'remote reset'
    }
    COI_FLAGS.update({idx: 'compatible reserved ({})'.format(idx) for idx in
                      range(COI_FLAG_COMPATIBLE_RESERVED_3, COI_FLAG_COMPATIBLE_RESERVED_31 + 1)})
    COI_FLAGS.update({idx: 'private reserved ({})'.format(idx) for idx in
                      range(COI_FLAG_PRIVATE_RESERVED_32, COI_FLAG_PRIVATE_RESERVED_127 + 1)})
```

Reply:
plz see 4142b27